src/Xrefcheck/CLI.hs

```diff
@@ -48,8 +51,21 @@ data Options = Options
     , oMode            :: VerifyMode
     , oVerbose         :: Bool
     , oShowProgressBar :: Bool
+    , oVerifyOptions   :: VerifyOptions
     }
 
+data VerifyOptions = VerifyOptions
+    { voNotScanned :: [FilePath]
+      -- ^ Folders, references in files of which we should not analyze.
```

Comment:
Here you say "Folders", below you say "Prefixes of files". It sounds a bit obscure and ambiguous, can you clarify the exact semantic? Note that in the issue I requested that it should be a folder or a file (in the former case everything in the folder is not scanned, in the latter case the file is not scanned).

Reply:
Mda.

I fixed to be "file prefix" as in config, in case of file it should be just filepath, in case of folder - filepath terminated with `/` (file prefix `src` would match both `src` and `src-files` folders, but `src/` would match only the former).

Actually, what do you think if I try to change it to glob pattern? That seems to me a more generic solution, but may be breaking for other projects. 

In either case, I think I will change those `FilePath`s to some custom type in order to make their semantics expressed in types.

Reply:
> I fixed to be "file prefix" as in config, in case of file it should be just filepath

Filepath or any prefix of filepath? Will `a` match everything that starts from `a` (folders and files)? Will empty string match everything (or is it not allowed)?

> Actually, what do you think if I try to change it to glob pattern? That seems to me a more generic solution, but may be breaking for other projects.

If `a` matches everything that starts from `a` then I think glob patterns are much saner. If `a` matches only `a`, using glob patterns seems to be a breaking change only in some weird corner cases (like when you have `*` in a file name). So in the former case I would do it because it's saner, in the latter case I would it unless it appears to be complicated.

Reply:
> Filepath or any prefix of filepath? Will a match everything that starts from a (folders and files)? Will empty string match everything (or is it not allowed)?

Yes, that's all true. Sounds not very good.

I'm going to try glob patterns then.